src/plugins/irc-events/message.js

```diff
@@ -41,13 +42,23 @@ module.exports = function(irc, network) {
 		let from;
 		let highlight = false;
 		let showInActive = false;
+		let shouldIgnore = false;
 		const self = data.nick === irc.user.nick;
 
+		// Check if the sender is in our ignore list
+		shouldIgnore = network.ignoreList.some(function(entry) {
```

Comment:
You can do `const shouldIgnore = ...` and move it just before `if` where this variable is relevant.